client-src/src/pages/inq-form1/inq-form1.html

```diff
@@ -57,79 +57,93 @@
           <ion-item>
             <ion-label>Education</ion-label>
             <ion-select formControlName="hQualification">
-              <ion-option *ngFor="let qualification of hQualifications" [value]=qualification.key>{{qualification.value}}</ion-option>
+              <ion-option *ngFor="let qualification of hQualifications" [value]=qualification>{{qualification}}</ion-option>
             </ion-select>
           </ion-item>
         </ion-col>
         <ion-col>
           <ion-item>
             <ion-label>Computer Knowledge</ion-label>
             <ion-select formControlName="computerKnowledge">
-              <ion-option *ngFor="let knowledge of computerKnowledge" [value]=knowledge.key>{{knowledge.value}}</ion-option>
+              <ion-option *ngFor="let knowledge of computerKnowledge" [value]=knowledge>{{knowledge}}</ion-option>
             </ion-select>
           </ion-item>
         </ion-col>
         <ion-col>
           <ion-item>
             <ion-label>Course Interested In</ion-label>
             <ion-select formControlName="areaOfInterest">
-              <ion-option *ngFor="let areaOfInterest of areasOfInterest" [value]=areaOfInterest.key>{{areaOfInterest.value}}</ion-option>
+              <ion-option *ngFor="let areaOfInterest of areasOfInterest" [value]=areaOfInterest>{{areaOfInterest}}</ion-option>
             </ion-select>
           </ion-item>
         </ion-col>
       </ion-row>
       <ion-item-divider color="light">Address
-        <span (click)="changeState()" class="changeState" float-right>Rajasthan&nbsp;&nbsp;
+        <!-- <span (click)="changeState()" class="changeState" float-right>Rajasthan&nbsp;&nbsp;
```

Comment:
Remove commented code.

Reply:
Backup code if this may be required later.

Reply:
Resolved in commit edf4f5f